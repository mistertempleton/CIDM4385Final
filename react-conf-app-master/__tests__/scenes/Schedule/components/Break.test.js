// @flow
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Break from '../../../../app/scenes/Schedule/components/Break';

describe('Schedule - Break', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Break status="present" important={false} title="" />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
