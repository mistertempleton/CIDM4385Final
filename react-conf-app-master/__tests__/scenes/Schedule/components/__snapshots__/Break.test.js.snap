exports[`Schedule - Break renders correctly 1`] = `
<View
  important={false}
  style={
    Array [
      Object {
        "alignItems": "stretch",
        "backgroundColor": "#fbfbfb",
        "borderBottomColor": "#CCC",
        "borderBottomWidth": 0.5,
        "flexDirection": "row",
      },
      undefined,
    ]
  }>
  <View
    style={
      Object {
        "backgroundColor": "#00A8D8",
        "width": 5,
      }
    } />
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#fbfbfb",
          "flexGrow": 1,
          "height": 44,
          "justifyContent": "center",
        },
        undefined,
      ]
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "color": "#383838",
            "fontSize": 14,
            "fontWeight": "300",
          },
          undefined,
        ]
      }>
       -
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontWeight": "500",
          }
        }>
         
        
      </Text>
    </Text>
  </View>
</View>
`;
